assets/js/components/legacy-setup/SetupUsingGCP.stories.js

```diff
@@ -64,6 +64,7 @@ Default.decorators = [
 	},
 ];
 Default.scenario = {
+	// eslint-disable-next-line sitekit/no-storybook-scenario-label
```

Comment:
This shouldn't be necessary since no changes were made to the file 🤔 

We should really avoid adding these ignores here and where needed, update the Story or scenario to comply with our convention instead.

Reply:
So for some reason, when I merged `develop` into my branch and tried to commit the merge, I was forced to do this otherwise the github hooks wouldn't let me commit. Since this issue has gone above estimate already, I didn't want to spend too much time on sorting these out and wanted to create another issue to do this.

Reply:
I thought skipping the pre-commit hooks would cause issues in the CI checks. But I think this is fine. Have reverted them all.